spec/commands/order/place_order_guest_spec.rb

```diff
@@ -1,41 +1,34 @@
 require 'commands/order/place_order_guest'
+require 'models/order'
+require 'spec_helper'
 
 RSpec.describe PlaceOrderGuest do
-  let (:event_data_from_will) { {user_id: "asdf", user_name: "Will", user_message: "order -james smith-: burger" } }
-
   it "save the order of a guest" do
-    place_order_guest = PlaceOrderGuest.new
-    place_order_guest.prepare(event_data_from_will)
-    place_order_guest.run
+    Helper.order_guest({name: "james smith", meal: "burger", from: "slack id"})
 
-     expect(Order.count).to eq(1)
+    expect(Order.last(:user_name => "james smith")).not_to be(nil)
   end
 
-  it "updates guest order when new order is placed for same guest" do
-    place_order_guest = PlaceOrderGuest.new
-    place_order_guest.prepare(event_data_from_will)
-    place_order_guest.run
+  it "a guest has a host" do
```

Comment:
> It a guest has a host.

Similar to a comment I left before. Try reading out the full line and ask yourself whether the sentence provides the real meaning behind the test.